homework/12_B/HW01points.py

```diff
@@ -0,0 +1,48 @@
+import math
+
+def interface(a):
+	if(len(a)==6):
+		return check_case(a[0],a[1],a[2],a[3],a[4],a[5])
+	else:
+		return 100
+
+def check_case(x1,y1,r1,x2,y2,r2):
+
+	if(x1==x2 and y1==y2 and r1==r2):
+		# съвпадат
+		return "same"
```

Comment:
Can you refactor the code to return enum instead of a string. Or at least move these hardcoded strings to global variables